### PR TITLE
[PLF-6615] Merge wars of Wiki Project

### PR DESCRIPTION
**Pull request by mtrabelsi**

### CODE REVIEW

**Comment by thomasdelhomenie** [comment]
The module wiki-extension and all its files should be deleted as well.

**Comment by mtrabelsi** [comment]
@thomasdelhomenie : PR updated, thanks.

**Comment by thomasdelhomenie** [comment]
The wiki extension dependency is also referenced in distribution project :
https://github.com/exoplatform/platform-public-distributions/blob/develop/plf-assemblies/src/main/resources/assemblies/plf-standalone-default-tomcat-distribution-content.xml#L606
https://github.com/exoplatform/platform-public-distributions/blob/develop/plf-dependencies/pom.xml#L769-L778
https://github.com/exoplatform/platform-private-distributions/blob/develop/plf-enterprise-jbosseap-ear/pom.xml#L346-L350
I am not sure there are others references. @mgreau am I correct ?

File: wiki-webapp/src/main/webapp/WEB-INF/conf/configuration.xml [hunk 1]
```diff
@@ -23,7 +23,6 @@
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
    xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
-
```
Comment (thomasdelhomenie) [inline wiki-webapp/src/main/webapp/WEB-INF/conf/configuration.xml]: Useless change

File: wiki-webapp/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 2]
```diff
@@ -14,8 +14,6 @@
 <gatein-resources xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_resources_1_3 http://www.gatein.org/xml/ns/gatein_resources_1_3"
    xmlns="http://www.gatein.org/xml/ns/gatein_resources_1_3">
 
- 
-
```
Comment (thomasdelhomenie) [inline wiki-webapp/src/main/webapp/WEB-INF/gatein-resources.xml]: Useless change

**Comment by thomasdelhomenie** [comment]
LGTM

**Pull request merged**